packages/framework/src/HydeServiceProvider.php

```diff
@@ -120,7 +120,7 @@ protected function registerModuleServiceProviders(): void
         $this->app->register(Modules\DataCollections\DataCollectionServiceProvider::class);
 
         if (env('DUSK_ENABLED', false)) {
-            $this->app->register(\Laravel\Dusk\DuskServiceProvider::class);
+            $this->app->register(\Hyde\Testing\Browser\DuskServiceProvider::class);
```

Comment:
The reason a custom service provider is needed is because the default Dusk provider expects the default Laravel routing which we don't have.